pkg/systemd/services/services.jsx

```diff
@@ -371,17 +365,6 @@ class ServicesPage extends React.Component {
                                         return;
                                     }
 
-                                    if (this.isTemplate(unit_id)) {
```

Comment:
Don't we have to filter out templates at least?

Reply:
Right, the UI looks ok because the subsequent LoadUnit will fail, but the Browser console is littered with `org.freedesktop.DBus.Error.InvalidArgs` warnings. We should either avoid the LoadUnit call, or suppress the warnings.

Reply:
good catch, fixing.